src/ConstantSum/ConstantSum.sol

```diff
@@ -89,27 +91,24 @@ contract ConstantSum is IStrategy {
         ConstantSumParams memory params =
             abi.decode(getPoolParams(poolId), (ConstantSumParams));
 
-        (uint256 startRx, uint256 startRy, uint256 startL) =
-            IDFMM(dfmm).getReservesAndLiquidity(poolId);
-
         (nextRx, nextRy, nextL) = abi.decode(data, (uint256, uint256, uint256));
```

Comment:
These are really `deltaRx`, `deltaRy` and `deltaL` right?

Reply:
It seems like these are still returned to `DFMM` as next reserves and liquidity

Reply:
@kinrezC is right, in this case the parameters are still the reserves, I think it's easier because we don't have to think about if the swap is x for y or vice versa.
@Autoparallel I think this is why you were talking about using `int256` yesterday? I understand how they could be useful here but I still think they are annoying to deal with.

Anyway, ff we want to use delta we can add a `isXForY` parameter.